cpp/arcticdb/storage/rocksdb/rocksdb_storage.cpp

```diff
@@ -87,11 +87,11 @@ RocksDBStorage::RocksDBStorage(const LibraryPath &library_path, OpenMode mode, c
 RocksDBStorage::~RocksDBStorage() {
     for (const auto& [key_type_name, handle]: handles_by_key_type_) {
         auto s = db_->DestroyColumnFamilyHandle(handle);
-        util::check(s.ok(), DEFAULT_ROCKSDB_NOT_OK_ERROR + s.ToString());
+        storage::check<ErrorCode::E_UNEXPECTED_ROCKSDB_ERROR>(s.ok(), DEFAULT_ROCKSDB_NOT_OK_ERROR + s.ToString());
```

Comment:
So the change from `util::check` to [`storage::check`](https://github.com/man-group/ArcticDB/blob/bb2278d845abef2b7e7101dc968f0974cf7bdde1/cpp/arcticdb/util/preconditions.hpp#L99) is to allow passing an error code. 👍